### PR TITLE
Update to React 18 and fix #26

### LINKED ISSUE #26: children is not assignable to <Link>
image
image

### PR DESCRIPTION
**Pull request by jeromeheissler**

### CODE REVIEW

**Comment by artemkhalygov** [comment]
@cvolant Not sure that you are the right person who should merge this pull request, but maybe you can merge it? Because now this library doesn't work on React 18.

File: package.json [hunk 1]
```diff
@@ -52,18 +52,18 @@
...
     "ts-jest": "^27.0.3",
     "typescript": "^4.3.4",
     "webpack": "^5.65.0",
     "yarn-or-npm": "^3.0.1"
   },
   "peerDependencies": {
-    "next": ">11.0.0",
-    "react": ">=16.8.0"
+    "next": ">12.0.0",
+    "react": ">=18.0.0"
```
Comment (cvolant) [inline package.json:67]: Is it possible to make it compatible with React 17, React 16.8 and Next 11?

**Review by cvolant — COMMENTED** [review]
Sorry for the delay. Lot of work + holidays.

**Comment by cvolant** [comment]
@jeromeheissler I merged it then allowed also next 11 and react 16.8 in an other commit. Is it ok for you?

**Comment by jeromeheissler** [comment]
I was also on holidays, it's perfect

**Pull request merged**